app/views/posts/index.html.erb

```diff
@@ -27,7 +28,7 @@
           <hr class="hr-divisor">
           <div class="user-post-comments-ctn d-flex col">
             <% if post.comments_counter > 0 %>
-              <% post.recent_comments.each do |comment| %>
+              <% post.recent_comments.includes([:author]).each do |comment| %>
                 <p> <%= comment.author.name %>: <%= comment.text %> </p>
               <% end %>
             <% else %>
```

Comment:
## OPTIONAL

- [ ]  Kindly make sure when the user clicks on the post details are rendered(post#show)

Reply:
Thank you @andersonlebon ! I just added that suggestion